src/pvrclient-argustv.h

```diff
@@ -122,6 +126,7 @@ class cPVRClientArgusTV : public kodi::addon::CInstancePVRClient
   std::string m_sBackendVersion;
   time_t m_BackendUTCoffset = 0;
   time_t m_BackendTime = 0;
+  std::string _streamFileName; // the name of the stream file
```

Comment:
`m_streamFilename`

Note that case change for the `n` also. But I'm not sure you need a member for this variable. It should be ok to just declare it in the function where it's used. Unless there is another reason to have it as a member?